apps/entity-service/src/entities/project-pitches.controller.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  BadRequestException,
+  Controller,
+  Get,
+  HttpStatus,
+  NotFoundException,
+  Param,
+  Query,
+  Request
+} from "@nestjs/common";
+import { buildJsonApi, getStableRequestQuery } from "@terramatch-microservices/common/util";
+import { ApiOperation } from "@nestjs/swagger";
+import { ExceptionResponse, JsonApiResponse } from "@terramatch-microservices/common/decorators";
+import { ProjectPitchService } from "./project-pitch.service";
+import { ProjectPitchDto } from "./dto/project-pitch.dto";
+import { ProjectPitchParamDto } from "./dto/project-pitch-param.dto";
+import { EntityQueryDto } from "./dto/entity-query.dto";
+
+@Controller("entities/v3/projectPitches")
+export class ProjectPitchesController {
+  constructor(private readonly projectPitchService: ProjectPitchService) {}
+
+  @Get()
+  @ApiOperation({
+    operationId: "ProjectPitchesIndex",
+    summary: "Get projects pitches."
+  })
+  @JsonApiResponse([{ data: ProjectPitchDto, pagination: "number" }])
+  @ExceptionResponse(BadRequestException, { description: "Param types invalid" })
+  @ExceptionResponse(NotFoundException, { description: "Records not found" })
+  async getPitches(@Request() { authenticatedUserId }, @Query() params: EntityQueryDto) {
```

Comment:
These methods and operationIds should follow the conventions set by the entity service. In this case, this controller should have `projectPitchIndex` and `projectPitchGet`, with the operation ID matching the function name. 

Additionally, a new project pitch policy is needed that matches the the read permissions in the policy from PHP. Let me know know if you need help wiring that up.

Let's also have a query DTO specific to this endpoint. The entity one is already overloaded, and is not meant to be a generic grab bag of whatever query params are needed for all entity service endpoints.

Finally, I see in the admin controller in PHP that there are lots of filtering and sorting controls needed for the admin dashboard that do not appear to be implemented here.